src/function/built_in_vector_operations.cpp

```diff
@@ -114,6 +118,10 @@ uint32_t BuiltInVectorOperations::getCastCost(
 
 uint32_t BuiltInVectorOperations::getTargetTypeCost(common::DataTypeID typeID) {
     switch (typeID) {
+    case common::INT32:
```

Comment:
Add bracket. Also, are we copying these magic numbers from duckdb? I think we need to add some tests or issue about

INT16 + INT32 will bind to function INT32 + INT32.

Reply:
added bracket